t/roundrobin.t

```diff
@@ -134,28 +133,16 @@ server2: 33334
 
             local rr = roundrobin:new(servers, true)
             local id = rr:find()
-            ngx.say(id)
-
-            math.randomseed(11175098)
-
-            local new_servers = {
-                ["server1"] = 1,
-                ["server2"] = 1,
-                ["server3"] = 1,
-                ["server4"] = 1,
-                ["server5"] = 1,
-                ["server6"] = 1,
-            }
 
-            rr:reinit(new_servers)
-            id = rr:find()
-            ngx.say(id)
+            local rr2 = roundrobin:new(servers, true)
```

Comment:
Better to use `reinit` like the original test.

Reply:
ok